src/hammer-tech/filters.py

```diff
@@ -93,6 +93,28 @@ def paths_func(lib: "Library") -> List[str]:
         return LibraryFilter.new("timing_lib", "CCS/NLDM timing lib (ASCII .lib)",
                                  paths_func=paths_func, is_file=True)
 
+    def timing_lib_nldm_filter(self, vts:List[str]=[]) -> LibraryFilter:
```

Comment:
```suggestion
    def timing_lib_nldm_filter(self, vts: Optional[List[str]] = None) -> LibraryFilter:
```

Could be dangerous to have mutable default values.

https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
rookie mistake